pkg/k8sapi/discovery.go

```diff
@@ -215,7 +215,14 @@ func (d *Controller) handlePodUpdate(key string) error {
 		d.workerPodsLock.Lock()
 		defer d.workerPodsLock.Unlock()
 
+		var containerID string
+
+		if len(pod.Status.ContainerStatuses) > 0 {
+			containerID = pod.Status.ContainerStatuses[0].ContainerID
```

Comment:
Could you add the same debug logging as when we fetched it from Docker?
```
log.Debugf("Found pod(%v)'s container ID: %v ", podName, containerID)
```

Reply:
updated!

Reply:
Ouch, that won't work, you haven't assigned `containerID` the value yet.

Reply:
note to self: no PRs before coffee